include/xlnt/utils/date.hpp

```diff
@@ -49,10 +51,15 @@ struct XLNT_API date
     static date from_number(int days_since_base_year, calendar base_date);
 
     /// <summary>
-    /// Constructs a data from a given year, month, and day.
+    /// Constructs a date from a given year, month, and day.
     /// </summary>
     date(int year_, int month_, int day_);
 
+    /// <summary>
+    /// Constructs an empty date (a call to is_null will return true).
+    /// </summary>
+    date() = default;
```

Comment:
I'm not sure if I like the default constructor here as it will result in a random integer as year/month/day.

Reply:
While other structs like [`std::chrono::year_month_day`](https://en.cppreference.com/w/cpp/chrono/year_month_day/year_month_day) do the same (unfortunately!), I agree that this is not nice, and it was not my intention. The year/month/day fields are now initialized to 0 by default, after my latest commit.